pyam/iiasa.py

```diff
@@ -73,74 +71,58 @@ def __init__(self, creds: str = None, auth_url: str = _AUTH_URL):
         ----------
         creds : pathlib.Path or str, optional
             Path to a file with authentication credentials
-        auth_url : str, optionl
+        auth_url : str, optional
             Url of the authentication service
         """
         self.client = httpx.Client(base_url=auth_url, timeout=10.0, http2=True)
-        self.access_token, self.refresh_token = None, None
 
         if creds is None:
             if DEFAULT_IIASA_CREDS.exists():
-                self.creds = _read_config(DEFAULT_IIASA_CREDS)
+                deprecation_warning(
+                    f"{IXMP4_LOGIN} and manually delete the file '{DEFAULT_IIASA_CREDS}'.",
+                    "Using a pyam-credentials file",
+                )
+                self.auth = _read_config(DEFAULT_IIASA_CREDS)
```

Comment:
That's what we usually do - first issue a warning for one or two release cycles, then remove (with a DeprecationWarning-error).

Otherwise, people will *have to* immediately update their credentials, which can be frustrating if at an inconvenient time...

Reply:
Fair point.